multiload/properties.c

```diff
@@ -639,6 +641,10 @@ fill_properties(GtkWidget *dialog, MultiloadApplet *ma)
 	add_color_selector (page, _("_Write"), "diskload-color1", ma);
 	add_color_selector (page, _("_Background"), "diskload-color2", ma);
 
+    gtk_widget_add_events (ma->notebook, GDK_SCROLL_MASK);
```

Comment:
Thanks for respecting our code styling preferences which is spaces. But here the whole file use tabs.
In this case it is better to follow code-styling from file.
Can you change that please?
Take care of tab size.
You can do that in a new commit if it is easier for you, to avoid breaking your repo.
I can squash that before merging.

Reply:
I meant only at this place. The other part (dialog_page_scroll_event_cb) is OK.

Reply:
Ok, i change tab identation to respect the file style and complete this PR squashing the commits...